rotkehlchen/api/rest.py

```diff
@@ -2921,8 +2927,17 @@ def get_asset_icon(
             )
             if response is not None:
                 return response
+            else:
+                return maybe_create_image_response(image_path=icon_path)
 
-        image_path = self.rotkehlchen.icon_manager.get_icon(asset)
+        image_path, processed = self.rotkehlchen.icon_manager.get_icon(asset)
```

Comment:
processed is not really the correct name to use here.

```suggestion
        image_path, launched_remote_query = self.rotkehlchen.icon_manager.get_icon(asset)
```